.changeset/many-eggs-deny.md

```diff
@@ -0,0 +1,7 @@
+---
+"wrangler": patch
+---
+
+fix: pages advanced mode usage
+
+This changes prevents the `pages dev` command from quitting in pages projects using `_worker.js`.
```

Comment:
Could you provide a bit more information about exactly what the scenario is that is broken here?

Reply:
As I understand it, if there is a `./functions` directory then this works correctly, yes?
So the scenario is that there is no `./functions` directory but there is a `singleWorkerScriptPath` (i.e. `--script-path`), which I guessis the `_worker.js` file that you describe above?

Reply:
Yes exactly, `./functions` works as it should, but when using a `singleWorkerScriptPath` wrangler won't get as far as launching miniflare and will quit.

I'll update the changeset to explain it a bit better.